solidity/core/test/inbox.test.ts

```diff
@@ -30,20 +30,19 @@ describe('Inbox', async () => {
   ];
 
   let inbox: TestInbox,
-    validatorManager: ValidatorManager,
     signer: SignerWithAddress,
-    fakeSigner: SignerWithAddress,
     abacusMessageSender: SignerWithAddress,
-    validator: Validator,
-    fakeValidator: Validator;
+    validatorManager: TestValidatorManager;
 
   before(async () => {
-    [signer, fakeSigner, abacusMessageSender] = await ethers.getSigners();
-    validator = await Validator.fromSigner(signer, remoteDomain);
-    fakeValidator = await Validator.fromSigner(fakeSigner, remoteDomain);
-    const validatorManagerFactory = new ValidatorManager__factory(signer);
-    validatorManager = await validatorManagerFactory.deploy();
-    await validatorManager.enrollValidator(remoteDomain, validator.address);
+    [signer, abacusMessageSender] = await ethers.getSigners();
+    // Inbox.initialize will ensure the validator manager is a contract.
```

Comment:
do we want to enforce the address implements the `ValidatorManager` interface?

Reply:
Do you mean to have some check in Inbox.sol that requires its validatorManager to implement certain functions?
If that's what you mean, in this new setup the Inbox is agnostic to the validatorManager's interface, so I don't think it can/should care what the validator manager looks like. Imo checking it's a contract is already pretty heavy handed but requiring certain functions would restrict the interface of future validator managers, which I believe we wanted to avoid to allow easy upgrades to future signature schemes